uiowa/uiowa
# Custom time-based cache tag

Resolves https://github.com/uiowa/uiowa/issues/4786

- Make sure the latest commit makes it out to uiowa DEV
- Checkout https://default.dev.drupal.uiowa.edu/about and see a cached event.
- Edit https://content.dev.drupal.uiowa.edu/node/82386/edit and change the title.
- Give it a moment and run `drush @default.dev pinv tag time:hours_3`
- Reload and see your updated title
- ??
- Profit
